Gemfile

```diff
@@ -36,4 +36,9 @@ group(:development, :test) do
   gem("aasm", require: false)
   gem("bcrypt", require: false)
   gem("xpath", require: false)
+
+  # net-smtp was removed from default gems in Ruby 3.1, but is used by the `mail` gem.
+  # So we need to add it as a dependency until `mail` is fixed:
+  # https://github.com/rails/rails/blob/0919aa97260ab8240150278d3b07a1547489e3fd/Gemfile#L178-L191
+  gem("net-smtp", "0.3.1", require: false)
```

Comment:
Which version of Rails are you testing against? If it is 7.0 you will not need this in a few minutes.

Reply:
We are still testing against Rails 6.1, yet. We will have to add Rails 7 support soon.